ruby/puppet

```diff
@@ -1 +1 @@
-Subproject commit d93f59152eeb30c292b5e134d39967e03e41ff61
+Subproject commit f6f511d4046b05566e5495defa59832fe58a1fa4
```

Comment:
Yep, this is the first commit in the core Ruby Puppet repo where the underlying `ClassInformationService` that this PR uses came in.  https://github.com/puppetlabs/puppet/commit/f6f511d4046b05566e5495defa59832fe58a1fa4.